contrib/nbfc-linux/files/nbfc_service

```diff
@@ -0,0 +1,8 @@
+# NoteBook FanControl service
+
+type            = bgprocess
+command         = /usr/bin/nbfc start
```

Comment:
this should be named 'nbfc' rather than 'nbfc_service' pretty sure, since the _service doesn't mean anything

Reply:
I'm just following how upstream named the service for other init systems

Reply:
I do think it is important to offer an interface that matches other distros, but sure, renamed